superset-frontend/stylesheets/superset.less

```diff
@@ -279,18 +279,20 @@ table.table-no-hover tr:hover {
   }
 }
 
-.nav.nav-tabs li .anchor-link-container {
-  top: 0;
-  right: -32px;
-}
-
 .dashboard-component.dashboard-component-header .anchor-link-container {
   .fa.fa-link {
     font-size: @font-size-l;
   }
 }
 
-.nav.nav-tabs li:hover,
+.ant-tabs-nav-list .ant-tabs-tab {
```

Comment:
Would much rather see this moved into Emotion, in the `Tabs` component, and using `gridUnit * 3` instead of `12`